Mathlib/ModelTheory/Substructures.lean

```diff
@@ -289,8 +289,22 @@ theorem lift_card_closure_le :
   refine lift_card_closure_le_card_term.trans (Term.card_le.trans ?_)
   rw [mk_sum, lift_umax.{w, u}]
 
+lemma closed_iff (s : Set M) :
+    (closure L).closed s ↔ ∀ {n}, ∀ f : L.Functions n, ClosedUnder f s := by
```

Comment:
`x.closed` is a set not a function
```suggestion
lemma mem_closed_iff (s : Set M) :
    s ∈ (closure L).closed ↔ ∀ {n}, ∀ f : L.Functions n, ClosedUnder f s := by
```
which hould also remove the need for the `change`